frontend/simple/views/UserProfileView.vue

```diff
@@ -1,26 +1,252 @@
 <template>
-  <vue-assistant :views="views" @done="done"></vue-assistant>
+  <section class="section full-screen">
+    <div class="columns">
+      <div class="column is-one-third"></div>
+      <div class="column" >
+        <form ref="ProfileForm"
+              name="ProfileForm"
+              @submit.prevent="save"
+              data-vv-scope="ProfileForm"
+        >
+          <p
+                  class="notification is-success has-text-centered"
+                  v-if="profileSaved"
+                  id="ProfileSaveSuccess"
+          >
+            <i class='notification-icon fa fa-check'></i>
+            <i18n>Profile saved successfully!</i18n>
+          </p>
+          <p
+                  class="notification is-danger has-text-centered"
+                  v-if="errorMsg"
+          >
+            <i class='notification-icon fa fa-exclamation-triangle'></i>
+            {{errorMsg}}
+          </p>
+          <div class="panel">
```

Comment:
Too narrow. This whole panel (and the group one below it) should be way wider, take up at least 80% of the screen, and on mobile take up even more.

+ It should clearly display the user's username somewhere.